UI/CUIControls.h

```diff
@@ -594,4 +595,58 @@ class RotatingGraphic : public GG::StaticGraphic {
     float   m_phase_offset;
 };
 
+class CUIToggleButton;
```

Comment:
Not sure what my alternative is here?  Without a forward, I will see multiple

    ...error: ‘CUIToggleButton’ does not name a type
    virtual void RenderUnchecked(const CUIToggleButton& button) const;
                                       ^

And the reverse for CUIToggleButtonRepresenter if I put CUIToggleButton first.